development.md

```
@@ -10,15 +10,17 @@ docker compose watch
 
 * Now you can open your browser and interact with these URLs:
 
-Frontend, built with Docker, with routes handled based on the path: http://localhost:5173
-
 Backend, JSON based web API based on OpenAPI: http://localhost:8000
 
 Automatic interactive documentation with Swagger UI (from the OpenAPI backend): http://localhost:8000/docs
 
+Alternative interactive documentation (ReDoc): http://localhost:8000/redoc
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)